src/patternfly/components/ProgressStepper/examples/ProgressStepper.md

```diff
@@ -0,0 +1,368 @@
+    {{#> progress-stepper-step-main}}
+      {{#> progress-stepper-step-title}}
+        Pending
+      {{/progress-stepper-step-title}}
+    {{/progress-stepper-step-main}}
+  {{/progress-stepper-step}}
+{{/progress-stepper}}
+```
+
+
+## Documentation
+
+### Overview
+The progress stepper is intended to show progress through a finite number of discrete steps.
+
+Add a modifier class to the progress stepper to change the orientation or alignment: `.pf-m-center`, `.pf-m-vertical`, or `.pf-m-compact`.
+
+Steps can be modified with `.pf-m-success`, `.pf-m-warning`, `.pf-m-danger`, and `.pf-m-info` to change their color. Use modifiers `.pf-m-pending` and `.pf-m-current` to indicate progress through the steps.
+### Accessibility
+| Attribute | Applied to | Outcome |
+| -- | -- | -- |
+| `aria-label="Completed step"` | `.pf-c-progress-stepper__step` |  Indicates the step is complete. |
+| `aria-label="Current step"` | `.pf-c-progress-stepper__step` |  Indicates the current step. |
+| `aria-label="Pending step"` | `.pf-c-progress-stepper__step` |  Indicates the step is pending. |
+| `aria-label="In process step"` | `.pf-c-progress-stepper__step` |  Indicates the step is in process. |
+| `aria-label="Step with issue"` | `.pf-c-progress-stepper__step` |  Indicates the step has a warning. |
+| `aria-label="Step with failure"` | `.pf-c-progress-stepper__step` |  Indicates the step has a failure. |
+| `aria-label="Information alert"` | `.pf-c-progress-stepper__step` |  Indicates the information alert. |
+| `aria-hidden="true"` | `.pf-c-progress-stepper__step-icon <i>` |  Hides icon for assistive technologies. **Required** |
+
+### Usage
+| Class | Applied to | Outcome |
+| -- | -- | -- |
+| `.pf-c-progress-stepper` | `<ol>` | Applies default progress stepper styling.  **Required** |
+| `.pf-c-progress-stepper__step` | `<li>` | Defines each step of the progress stepper. **Required** |
+| `.progress-stepper-step-connector` | `<div>` | Creates the connecting line between steps **Required** |
+| `.progress-stepper-step-icon` | `<span>` | Creates the step node and contains the icon designating the type of step. This element is required, but may be empty if no icon is used for the step. **Required** |
+| `.progress-stepper-step-main` | `<div>` | Contains the main text content of the step. This element is required, but may be empty if title and description are not used. **Required** |
+| `.progress-stepper-step-title` | `<div>` | Contains the title of the step. |
+| `.progress-stepper-step-description` | `<div>` | Contains the description of the step. |
+
+### Modifiers
+| Class | Applied to | Outcome |
+| -- | -- | -- |
+| `.pf-m-center`| `.pf-c-progress-stepper` | Centers the text content of each step. |
+| `.pf-m-vertical`| `.pf-c-progress-stepper` | Changes the orientation to show steps vertically. |
+| `.pf-m-compact`| `.pf-c-progress-stepper` | Changes the styling to be compact. The title of the current step is shown above the steps. |
```

Comment:
Up to you for how to document this, but leaving off the details of what all the modifier does (including the success, error, etc states below) and just saying it "modifies for [compact, success, etc] styling" is less error prone for having to remember to update the docs when we make a change to the modifier. Most of our modifiers are documented pretty generically that way.